### PR TITLE
Fix use latest protocol

### PR DESCRIPTION
**Pull request by milanzlicic**

### CODE REVIEW

File: src/repository/SQLiteDeviceRepository.h [hunk 1]
```diff
@@ -17,8 +17,8 @@
 #ifndef DEVICEREPOSITORYIMPL_H
 #define DEVICEREPOSITORYIMPL_H
 
-#include <Poco/Data/Session.h>
 #include "repository/DeviceRepository.h"
+#include <Poco/Data/Session.h>
```
Comment (srdjastankovic) [inline src/repository/SQLiteDeviceRepository.h:21]: Have some impact?

**Review by srdjastankovic — COMMENTED** [review]
@milanzlicic you include new version of WolkSDK, I don't have to look at it? We did it separatly?

File: src/repository/SQLiteDeviceRepository.h (hunk 1 above)
Comment (milanzlicic) [reply src/repository/SQLiteDeviceRepository.h:21]: Just sorting done by format

**Comment by milanzlicic** [comment]
> @milanzlicic you include new version of WolkSDK, I don't have to look at it? We did it separatly?

Yes

**Review by srdjastankovic — APPROVED** [review]

**Pull request merged**